site/app/templates/course/CourseMaterials.twig

```diff
@@ -61,7 +61,7 @@
             <div class="file-viewer">
                 {% if course_material.isLink %}
                     <i class="fas fa-link" style="vertical-align: text-bottom;"></i>
-                    <a href="{{ course_material.getUrl }}" onclick="markViewed([{{ course_material.getId }}])" onauxclick="markViewed([{{ course_material.getId }}])">{{ course_material.getUrlTitle }}</a>
+                    <button href="{{ course_material.getUrl }}" onclick="markViewed([{{ course_material.getId }}])" onauxclick="markViewed([{{ course_material.getId }}])">{{ course_material.getUrlTitle }}</button>
```

Comment:
This should not be converted to a `button`, instructors may also post links in the course materials page - this is what is causing the cypress e2e error since the link is no longer functional.

<img width="1177" alt="image" src="https://user-images.githubusercontent.com/12129065/170621914-34c811a8-de33-4de8-9f97-9dcf96340a7b.png">